scripts/monitoring/cron-send-docker-metrics.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python2
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+'''
+    docker info data gatherer
+'''
+
+# Adding the ignore because it does not like the naming of the script
+# to be different than the class name
+# pylint: disable=invalid-name
+
+
+from docker import AutoVersionClient
+from docker.errors import DockerException
+from openshift_tools.monitoring.zagg_sender import ZaggSender
+from openshift_tools.timeout import TimeoutException
+from openshift_tools.monitoring.dockerutil import DockerUtil
+import json
+
+if __name__ == "__main__":
+    keys = None
+    zs = ZaggSender()
+    try:
+        cli = AutoVersionClient(base_url='unix://var/run/docker.sock')
+        du = DockerUtil(cli)
+        du_dds = du.get_disk_usage()
```

Comment:
Does this still timeout?

I don't see where it's timing out now.

Should we use a "with timeout(15):" here?